### DISCUSSION TITLE
Trying to hide a column on small devices

### DISCUSSION THREAD

**Discussion by MarcoCanada-nbcuni** [tailwindlabs/tailwindcss#7821] [Help]
I am trying to hide a table column on small devices. Since tailwind is mobile-first (or small device first), I wrote the code below
```<th class="hidden md:table-cell"> Purchase Date</th>```
but that doesn't work. 
However if I do the opposite it does work (hides the column on medium and above devices)
```<th class="table-cell md:hidden"> Purchase Date</th>```

**Comment by alexanderbluhm** [comment]
@MarcoCanada-nbcuni I am not exactly sure why but it works fine for me if I use valid HTML for the `th`.
Maybe if otherwise some default (preflight) styles are applied automatically.
https://play.tailwindcss.com/dsEXBk0sab?size=780x720
